src/test/java/ru/training/at/hw3/page/HomePage.java

```diff
@@ -0,0 +1,146 @@
+package ru.training.at.hw3.page;
+
+import ru.training.at.hw3.utils.Prop;
+import ru.training.at.hw3.utils.WaitActions;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import javax.inject.Inject;
+import java.util.List;
+
+public class HomePage {
+
+    private WaitActions waitActions;
+    private WebDriver webDriver;
+    private WebElement webElement;
```

Comment:
Code duplication from **src/test/java/ru/training/at/hw3/page/DifferentElementsPage.java**

Reply:
extracted to BasePage, no duplication any more